feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt

```diff
@@ -0,0 +1,223 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.iteneum.apartment.R
+import com.iteneum.designsystem.components.LPTitleLarge
+import com.iteneum.designsystem.components.LpGenericChip
+import com.iteneum.designsystem.components.LpOutlinedButton
+import com.iteneum.designsystem.components.LpPaymentsRentCard
+import java.sql.Timestamp
+
+/**
+ * [ApartmentView] The apartment view is the screen where the user watches your payments, repairs, information
+ * [ApartmentContainer] This section is where all the sections are, because all the sections are divided each one
+ * [TitleApartmentScreen] This section is where only to be the tittle of the screen, in this case: "Apartment"
+ * [PaymentsSection] All the params for this sections is for show the current status, date, month and their account,
+ * in this section the user can their account
+ * [RepairsSection] and [RepairStatusRow], work together for show and open a new repairs application with the button,
+ * the user can watch the Open, In Progress and closed repairs status
+ * [InformationSection] is where the user have a their disposable some buttons for leasing documents, apartment photos and current status
+ *
+ * @author Usiel Filiberto Garcia Jimenez
+ */
+@Composable
+@Preview
+fun ApartmentView() {
+    ApartmentContainer()
+}
+
+@Composable
+fun ApartmentContainer() {
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                start = LeasePertTheme.sizes.midSmallSize,
+                end = LeasePertTheme.sizes.midSmallSize
+            )
+            .verticalScroll(state = rememberScrollState())
+    ) {
+        Column {
+            TitleApartmentScreen()
+            PaymentsSection(Modifier, "Junary", 990.00, Timestamp(23435))
+            RepairsSection()
+            InformationSection()
+        }
+    }
+}
+
+@Composable
+fun TitleApartmentScreen() {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(
+                bottom = LeasePertTheme.sizes.extraSize18,
+                top = LeasePertTheme.sizes.minorRegularSize
+            )
+    ) {
+        LPTitleLarge(
+            label = stringResource(
+                id = R.string.apartment
+            ),
+            color = MaterialTheme.colorScheme.onPrimaryContainer
+        )
+    }
+}
+
+@Composable
+fun PaymentsSection(modifier: Modifier, month: String, quantityToPay: Double, date: Timestamp) {
+    Column() {
+        Row(
+            modifier = modifier
+                .fillMaxWidth()
+                .padding(bottom = LeasePertTheme.sizes.regularSize),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = stringResource(id = R.string.payments),
```

Comment:
Please check the fontWeight for each section title

Reply:
Done